spec.html

```diff
@@ -3837,7 +3837,7 @@ <h1>Well-Known Intrinsic Objects</h1>
               <td>
               </td>
               <td>
-                The prototype of RegExp String Iterator objects (<emu-xref href="#sec-regexp-string-iterator-objects"></emu-xref>)
+                The prototype of RegExp String iterator objects (<emu-xref href="#sec-regexp-string-iterator-objects"></emu-xref>)
```

Comment:
I would prefer to leave `Iterator` capitalized when it's part of a name like this. "RegExp String Iterator" is a proper noun.

Reply:
I'm fine with that. Same with Array/String/Map/Set I'm assuming?

Reply:
Yeah.

Reply:
Done.